### PR TITLE
adding some markup tags to documentation

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A  |
|---          |---|
|Bug Fix?     |n  |
|New Feature? |n  |
|BC Breaks?   |n  |
|Deprecations?|n  |
|Tests Pass?  |n  |
|Fixed Tickets|   |
|License      |MIT|
|Doc PR       |   |
                   

 Sent using [Gush](https://github.com/gushphp/gush)

### CODE REVIEW

File: Resources/doc/index.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function registerBundles()
 
 ### Step 3: Register the routing definition
 
-``` yml
```
Comment (willdurand) [inline Resources/doc/index.md]: Useless change

File: Resources/doc/index.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ Usage
 
 Add these two lines in your layout:
 
-```
+```html
```
Comment (willdurand) [inline Resources/doc/index.md]: jinja?

File: Resources/doc/index.md [hunk 3]
```diff
@@ -98,11 +98,13 @@ my_route_to_expose_with_defaults:
 
 Or using annotations:
 
+```php
     # src/Acme/DemoBundle/Controller/DefaultController.php
```
Comment (willdurand) [inline Resources/doc/index.md]: Fix indentation please, and change the sharp to `//` please

File: Resources/doc/index.md (hunk 1 above)
Comment (cordoval) [inline Resources/doc/index.md]: just to make it consistent to be either yml everywhere or yaml which is more used and correct

File: Resources/doc/index.md (hunk 2 above)
Comment (cordoval) [inline Resources/doc/index.md]: true, i will switch

File: Resources/doc/index.md (hunk 3 above)
Comment (cordoval) [inline Resources/doc/index.md]: good point

**Comment by cordoval** [comment]
done

**Comment by willdurand** [comment]
Thanks! Could you squash your two commits?

**Comment by cordoval** [comment]
all good after a ```gush p:squash 175```

**Comment by willdurand** [comment]
Thanks Luis

**Pull request merged**